dart/test/scope_test.dart

```diff
@@ -222,6 +222,42 @@ void main() {
     expect(sut.eventProcessors.length, 0);
   });
 
+  test('apply context to event', () {
+    final user = User(
+      id: '800',
+      username: 'first-user',
+      email: 'first@user.lan',
+      ipAddress: '127.0.0.1',
+      extras: <String, String>{'first-sign-in': '2020-01-01'},
+    );
+    final breadcrumb = Breadcrumb(message: 'Authenticated');
+
+    final event = SentryEvent();
+    final scope = Scope(SentryOptions())
+      ..user = user
+      ..fingerprint = ['example-dart']
+      ..addBreadcrumb(breadcrumb)
+      ..transaction = '/example/app'
+      ..level = SentryLevel.warning
+      ..setTag('build', '579')
+      ..setExtra('company-name', 'Dart Inc')
+      ..setContexts(key: 'theme', value: 'material')
+      ..addEventProcessor(
+        (event, hint) => event..tags.addAll({'page-locale': 'en-us'}),
+      );
```

Comment:
please add a few values to the key/value objects to test the precedence over others, eg
a context key on the event cant be replaced by a value of the same key on the scope.

for the level, this is actually the opposite, the scope level has precedence over the event level

Reply:
✅